Groups/GroupAttendanceMulti.ascx.cs

```diff
@@ -352,6 +365,58 @@ protected void cbAttendee_CheckedChanged( object sender, EventArgs e )
             }
         }
 
+        /// <summary>
+        /// Handles the SelectedIndexChanged event of the ddlAddPersonGroup control.
+        /// </summary>
+        /// <param name="sender">The source of the event.</param>
+        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
+        protected void ddlAddPersonGroup_SelectedIndexChanged( object sender, EventArgs e )
+        {
+            if ( ddlAddPersonGroup.SelectedValue != "" )
```

Comment:
Consider using this event to set the ppAddPerson.PersonName to something group specific. Maybe similar to "Select Person as GROUPNAME attendee".